engine/flink/components/table/src/main/scala/pl/touk/nussknacker/engine/flink/table/utils/RowConversions.scala

```diff
@@ -65,4 +67,37 @@ object RowConversions {
 
   }
 
+  private implicit def contextIdCodec: Codec[ContextId] =
```

Comment:
I use the circe codec here, and keep the contextId in the single String field, as before. The reason is that we may need to further modify ContextId representation, without making multiple (very) problematic changes in the Flink serialization. But at the same time I am not sure if we are supposed to use circe in this context - whether it will have impact on performance.